### DISCUSSION TITLE
Skip "sub only" VODs if not subbed

### DISCUSSION THREAD

**Discussion by bonny1992** [Zibbp/ganymede#82] [Q&A] [answered]
Heya!
Is there a way to automatically skip a VOD if I'm not actively subbed to a specific channel? I tried looking into the channel settings to no avail.

Right now it just gives "Error" and the entire queue is stuck because of that, instead of erroring and continuing.

Thank you!

EDIT:
I forgot to say that at the moment I just deleted the stuck elements from the queue (and the element is still present on the channel page with the "Archiving" text over it)

**Comment by Zibbp** [comment, marked as answer]
There is currently not a way to skip sub-only vods, although it could be added. Are all of the channel's vods sub-only, or just one? If you could provide the channel I can use it for testing while developing that option as I don't know of any sub-only channels or vods.

> I forgot to say that at the moment I just deleted the stuck elements from the queue (and the element is still present on the channel page with the "Archiving" text over it)

You will need to delete the VOD entry in Admin > VODs along with the queue item, then it should disappear. Note that is does not delete any files, it simply removes the entry from the database.

**Reply by bonny1992** [reply]
Thanks for the fast reply Zibbp!

As far as I know, this streamer does "subs-only" streams only once in a while. So no, out of all the VODs of this streamer only 3 are subs-only.

Sure, the channel is this `https://www.twitch.tv/ilgattosultubo`.

**Reply by Zibbp** [reply]
@bonny1992 [v1.2.0](https://github.com/Zibbp/ganymede/releases/tag/v1.2.0) has been released which adds a toggle for downloading sub-only videos when adding or editing a watched channel. The setting has a default value of false, skipping sub-only videos when archiving a channel's videos. Hope this helps, let me know if you find any issues with this feature.

**Reply by bonny1992** [reply]
> @bonny1992 [v1.2.0](https://github.com/Zibbp/ganymede/releases/tag/v1.2.0) has been released which adds a toggle for downloading sub-only videos when adding or editing a watched channel. The setting has a default value of false, skipping sub-only videos when archiving a channel's videos. Hope this helps, let me know if you find any issues with this feature.

Well @Zibbp honestly I wasn't expecting a fix THAT fast haha.
I updated the container and I can see the toggle in the "Watched Channel" edit or new page.

Thank you, very much!
